sidebars.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
   serverSidebar: [
-    'server/introduction',
+    {
+      type: 'doc',
+      label: 'Temporal Server', 
+      id: 'server/introduction',
+    },
```

Comment:
I don't think this will have any effect.
These are defined in the file already.